deployments/docker-compose.yaml

```diff
@@ -0,0 +1,70 @@
+version: '3'
+services:
+  redis-node-0:
+    build:
+      context: ../
+      dockerfile: ./build/RedisClusterDockerfile
+    image: redis-cluster:6.2.1-alpine
+    networks:
+      podium:
+        aliases:
+          - redis-node-0
+  redis-node-1:
+    build:
+      context: ../
+      dockerfile: ./build/RedisClusterDockerfile
+    image: redis-cluster:6.2.1-alpine
+    networks:
+      podium:
+        aliases:
+          - redis-node-1
+  redis-node-2:
+    build:
+      context: ../
+      dockerfile: ./build/RedisClusterDockerfile
+    image: redis-cluster:6.2.1-alpine
+    networks:
+      podium:
+        aliases:
+          - redis-node-2
+
+  initialize-cluster:
```

Comment:
Why don't you use a noun here? Docker containers usually are named after nouns. For example: `redis-cluster`

Reply:
How this container is just a sidecar I prefer a name, as an action to be executed. Why do you think?